plugins/catalog-backend/migrations/20200511113813_init.js

```diff
@@ -36,7 +36,7 @@ exports.up = async function up(knex) {
           .comment('Auto-generated ID of the location');
         table.string('type').notNullable().comment('The type of location');
         table
-          .string('target')
+          .text('target')
```

Comment:
Note that this is an ancient migration. People who have adopted the catalog probably ran this migration years ago and won't try again. We generally don't modify old migrations for this reason - what you want to do is rather to add a new migration that alters the column type. Remember to update the `tests/migrations.test.ts` accordingly afterwards, testing both forward and backward migrations.